### PR TITLE
Adds a Hide/Show AOOC verb.

### PR DESCRIPTION
**Pull request by Yonnimer**
Does exactly what it says in the title. Only mods/admins/ccia get the verb, given only they can see it as non-antags. Normal antags won't' get access to the verb.

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

File: code/modules/admin/verbs/antag-ooc.dm [hunk 1]
```diff
@@ -14,6 +14,10 @@
 	if(!msg)
 		return
 
+	if(!(prefs.toggles & CHAT_AOOC))
+		to_chat(src, "<span class='warning'>You have AOOC muted.</span>")
```
Comment (mikomyazaki) [inline code/modules/admin/verbs/antag-ooc.dm]:
```suggestion
		to_chat(src, span("warning","You have AOOC muted."))
```

**Comment by Yonnimer** [comment]
The main point of this is so staff have a way to turn off wood, given the only way currently is to deadmin. This includes ccia, mods, and admins. I’m all for a better method to implement this, this was just the way I could.

**Comment by Yonnimer** [comment]
Updated so only staff get the verb now.

File: code/modules/admin/admin_verbs.dm [hunk 2]
```diff
@@ -98,7 +98,8 @@ var/list/admin_verbs_admin = list(
 	/client/proc/clear_toxins,
 	/client/proc/wipe_ai,	// allow admins to force-wipe AIs
 	/client/proc/fix_player_list,
-	/client/proc/reset_openturf
+	/client/proc/reset_openturf,
+	/client/proc/listen_Aooc
```
Comment (NonQueueingMatt) [inline code/modules/admin/admin_verbs.dm]: Rename this to `aooc`.

File: code/modules/admin/admin_verbs.dm [hunk 3]
```diff
@@ -781,6 +784,16 @@ var/list/admin_verbs_cciaa = list(
...
 
+/client/proc/listen_Aooc()
+	set name = "Show/Hide AOOC"
+	set category = "Preferences"
+	
+	prefs.toggles ^= CHAT_AOOC
+	if (prefs.toggles & CHAT_AOOC)
+		to_chat(usr, "You now will see messages in aooc")
+	else
+		to_chat(usr, "You now won't see messages in aooc")
```
Comment (NonQueueingMatt) [inline code/modules/admin/admin_verbs.dm]:
```suggestion
		to_chat(usr, "You will no longer see AOOC messages.")
```

File: code/modules/admin/admin_verbs.dm [hunk 4]
```diff
@@ -781,6 +784,16 @@ var/list/admin_verbs_cciaa = list(
 		to_chat(src, "<span class='interface'>You now have the keys to control the planet, or atleast a small space station</span>")
 		verbs -= /client/proc/readmin_self
 
+/client/proc/listen_Aooc()
+	set name = "Show/Hide AOOC"
+	set category = "Preferences"
+	
+	prefs.toggles ^= CHAT_AOOC
+	if (prefs.toggles & CHAT_AOOC)
+		to_chat(usr, "You now will see messages in aooc")
```
Comment (NonQueueingMatt) [inline code/modules/admin/admin_verbs.dm]:
```suggestion
		to_chat(usr, "You now will see all messages in AOOC.")
```

File: code/modules/client/preferences_toggles.dm [hunk 5]
```diff
@@ -120,8 +120,8 @@
 
 	to_chat(src, "You will [(prefs.toggles & CHAT_LOOC) ? "now" : "no longer"] see messages on the LOOC channel.")
 	feedback_add_details("admin_verb","TLOOC") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
-
-
+	
+
```
Comment (NonQueueingMatt) [inline code/modules/client/preferences_toggles.dm]: whitespace, get rid of it

File: code/modules/admin/admin_verbs.dm [hunk 6]
```diff
@@ -781,6 +784,16 @@ var/list/admin_verbs_cciaa = list(
 		to_chat(src, "<span class='interface'>You now have the keys to control the planet, or atleast a small space station</span>")
 		verbs -= /client/proc/readmin_self
 
+/client/proc/listen_Aooc()
+	set name = "Show/Hide AOOC"
+	set category = "Preferences"
+
```
Comment (NonQueueingMatt) [inline code/modules/admin/admin_verbs.dm]: Add a permissions check here.

File: code/modules/admin/admin_verbs.dm (hunk 3 above)
Comment (NonQueueingMatt) [reply code/modules/admin/admin_verbs.dm]: This should also be logged, by the by.

Comment (NonQueueingMatt) [reply code/modules/admin/admin_verbs.dm]: And you should also probably add a span class to this.

File: code/modules/admin/admin_verbs.dm (hunk 6 above)
Comment (Yonnimer) [reply code/modules/admin/admin_verbs.dm]: Done.

File: code/modules/client/preferences_toggles.dm (hunk 5 above)
Comment (Yonnimer) [reply code/modules/client/preferences_toggles.dm]: removed.

File: code/modules/admin/admin_verbs.dm (hunk 2 above)
Comment (Yonnimer) [reply code/modules/admin/admin_verbs.dm]: renamed

File: code/modules/admin/admin_verbs.dm [hunk 7]
```diff
@@ -781,6 +784,18 @@ var/list/admin_verbs_cciaa = list(
 		to_chat(src, "<span class='interface'>You now have the keys to control the planet, or atleast a small space station</span>")
 		verbs -= /client/proc/readmin_self
 
+/client/proc/listen_aooc()
+	set name = "Show/Hide AOOC"
```
Comment (NonQueueingMatt) [inline code/modules/admin/admin_verbs.dm]: No single line code please. Put the return below.

File: code/modules/admin/admin_verbs.dm [hunk 8]
```diff
@@ -781,6 +784,18 @@ var/list/admin_verbs_cciaa = list(
...
+	set category = "Preferences"
+	
+	if(!check_rights(R_ADMIN|R_MOD|R_CCIAA))	return
```
Comment (skull132) [inline code/modules/admin/admin_verbs.dm]:
```suggestion
	if(!check_rights(R_ADMIN|R_MOD|R_CCIAA))
		return
```

**Pull request merged**